backend/src/app/modules/course/course.service.ts

```
@@ -195,13 +200,156 @@ const checkEnrollment = async (courseId: string, user: JwtPayload) => {
+    })
+
+    const progress = await prisma.courseProgress.findUnique({
+        where: {
+            studentId_courseId: {
+                courseId: courseId,
+                studentId: user.userId
+            }
+        }
+    })
+
+    let result: any = course
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)